docs/usage/executor/index.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
this is moved from https://github.com/kubeshop/botkube-docs/blob/main/docs/usage/executing-commands.md

It needs to be adjusted after merging https://github.com/kubeshop/botkube/pull/905